src/recon.rs

```diff
@@ -2628,11 +2628,10 @@ pub(crate) unsafe fn rav1d_recon_b_intra<BD: BitDepth>(
                             ),
                         ]);
                         let top_sb_edge_slice = if t.b.y & f.sb_step - 1 == 0 {
-                            let mut top_sb_edge: *const BD::Pixel =
-                                f.ipred_edge[0] as *mut BD::Pixel;
                             let sby = t.b.y >> f.sb_shift;
-                            top_sb_edge = top_sb_edge.offset((f.sb128w * 128 * (sby - 1)) as isize);
-                            Some(slice::from_raw_parts(top_sb_edge, f.sb128w as usize * 128))
+                            let offset = f.ipred_edge_off as isize * 0
```

Comment:
`f.ipred_edge_off as isize * 0 +` doesn't do anything.

Reply:
Oh, I guess this is just here for parity with below where it's not `0`?

Reply:
Yup, the original logic was doing `ipred_edge[0]`, which changed to `ipred_edge_off * 0`. I wanted to keep that in there for parity with the other branches.